input/blog/posts/net-foundation-february-2019-update.md

```diff
@@ -23,7 +23,7 @@ This month's newsletter includes:</p>
 
 <ul>
 <li><u><a href="/blog/2019/01/14/why-you-should-join-part-1-join-as-a-voting-member">Why You Should Join: Part 1 - Voting Member</a></u></li>
-<li><a href="/blog/2019/01/16/why-you-should-join-part-2---corporate-sponsor">Why You Should Join: Part 2 - Corporate Sponsor</a></li>
+<li><a href="/blog/2019/01/16/why-you-should-join-part-2-corporate-sponsor">Why You Should Join: Part 2 - Corporate Sponsor</a></li>
```

Comment:
Should I rename the original markdown file, too?

Reply:
Thanks. Yes, can you please rename the original file and then we can merge the changes?

Reply:
@clairernovotny I renamed the file and rebased onto current master.